epan/dissectors/packet-ippusb.c

```diff
@@ -25,66 +25,104 @@
 #include "packet-usb.h"
 #include <stdio.h>
 
+#define HTTP 0
+#define IPP 1
+
 void proto_register_ippusb(void);
 void proto_reg_handoff_ippusb(void);
+static int is_http_header(int first_linelen, const guchar *first_line); 
 
 static int proto_ippusb = -1;
```

Comment:
Is there a reason why we use gint for some and int for others?

Reply:
It looks like the other packet dissectors do this too (not sure why).  I think gint is defined in glib.h

---
In reply to: [448664825](https://github.com/tjamhar/wireshark/pull/2#discussion_r448664825) [](ancestors = 448664825)

Reply:
The developer guide suggests using gint since is doesn't change across systems in size and there is a difference in uses because I was following the convention from other dissectors. I will change them to all gints.

---
In reply to: [448664825](https://github.com/tjamhar/wireshark/pull/2#discussion_r448664825) [](ancestors = 448664825)